sklearn/kernel_ridge/tests/test_krc.py

```diff
@@ -0,0 +1,228 @@
+"""
+Testing for Kernel Ridge Classification
+(sklearn.kernel_ridge)
+
+Authors: Carlos Perales <sir.perales@gmail.com>
+License: BSD 3 clause
+
+"""
+import numpy as np
+from numpy.testing import assert_array_equal, assert_array_almost_equal
+from numpy.testing import assert_almost_equal
+from scipy import sparse
+from sklearn import datasets, base
+from sklearn.utils import check_random_state
+from sklearn.utils.testing import assert_equal, assert_true
+from sklearn.utils.testing import assert_greater
+from sklearn.utils.testing import assert_raises_regexp
+from sklearn.utils.testing import assert_raises
+from sklearn.externals import six
+from sklearn.utils.estimator_checks import check_estimator
+from sklearn.kernel_ridge import KRC
+
+# toy sample
+X = [[-2, -1], [-1, -1], [-1, -2], [1, 1], [1, 2], [2, 1]]
+Y = [1, 1, 1, 2, 2, 2]
+T = [[-1, -1], [2, 2], [3, 2]]
+true_result = [1, 2, 2]
+
+# also load the iris dataset
+iris = datasets.load_iris()
+rng = check_random_state(42)
+perm = rng.permutation(iris.target.size)
+iris.data = iris.data[perm]
+iris.target = iris.target[perm]
+
+
+def test_check_estimator():
+    check_estimator(KRC)
+
+
+def test_krc_parameters():
+    # Test parameters.
+    clf = KRC(kernel='linear').fit(X, Y)
+    assert_array_equal(clf.predict(X), Y)
+
+
+def test_iris():
+    # Check consistency on dataset iris.
+
+    # shuffle the dataset so that labels are not ordered
+    for k in ('linear', 'rbf'):
+        clf = KRC(kernel=k).fit(iris.data, iris.target)
+        assert_greater(np.mean(clf.predict(iris.data) == iris.target), 0.75)
+
+    assert_array_equal(clf.classes_, np.sort(clf.classes_))
```

Comment:
This kind of thing is tested by common tests. You do not need to.

Reply:
Same that ( https://github.com/scikit-learn/scikit-learn/pull/10633#discussion_r168151047 ), I'll remove it.